### PR TITLE
Add recurring feature to normal plugin

### PR DESCRIPTION
**Pull request by wankimmy**

### CODE REVIEW

File: WooCommerce/woocommerce_rms_normal/class-gateway.php [hunk 1]
```diff
@@ -25,6 +25,7 @@ public function __construct() {
     $this->verify_key = $this->settings['verify_key'];
     $this->secret_key = $this->settings['secret_key'];
     $this->account_type = $this->settings['account_type'];
+    $this->recurring = $this->settings['recurring'];
     
     // Define hostname based on account_type
     $this->url = ($this->get_option('account_type')=='1') ? "https://pay.merchant.razer.com/" : "https://sandbox.merchant.razer.com/" ;
```
Comment (wankimmy) [inline WooCommerce/woocommerce_rms_normal/class-gateway.php]: change pay.merchant.razer.com to pay.fiuu.com

File: WooCommerce/woocommerce_rms_normal/class-gateway.php [hunk 2]
```diff
@@ -3,10 +3,10 @@ class WC_Molpay_Gateway extends WC_Payment_Gateway {
   
   // Constructor method
   public function __construct() {
-    global $woocommerce;
+    global $woocommerce, $post;
 
     $this->id                 = 'wcmolpay';
-    $this->icon = plugins_url( 'images/Fiuu_Logo-01.png', __FILE__ );
+    $this->icon = plugins_url( 'images/Fiuu_Logo-03.png', __FILE__ );
```
Comment (wankimmy) [inline WooCommerce/woocommerce_rms_normal/class-gateway.php]: upload the image 03 file

File: WooCommerce/woocommerce_rms_normal/class-gateway.php [hunk 3]
```diff
@@ -376,13 +434,156 @@ function check_molpay_response_callback() {
           $status = $_POST['status'];
           if ( !$verifyresult )
               $status = "-1";
-          
-          $WCOrderId = $this->get_WCOrderIdByOrderId($_POST['orderid']);
+
+		    $extraP = isset($_POST['extraP']) ? $_POST['extraP'] : '';
+
+			if (isset($_POST['extraP']) && !empty($_POST['extraP']) && ($this->recurring == 'yes')) {
```
Comment (wankimmy) [inline WooCommerce/woocommerce_rms_normal/class-gateway.php]: rearrange the code, make it pretty

File: WooCommerce/woocommerce_rms_normal/class-gateway.php (hunk 2 above)
Comment (syahruddin-syahrir) [reply WooCommerce/woocommerce_rms_normal/class-gateway.php]: image

file uploaded

File: WooCommerce/woocommerce_rms_normal/class-gateway.php (hunk 1 above)
Comment (syahruddin-syahrir) [reply WooCommerce/woocommerce_rms_normal/class-gateway.php]: image

File: WooCommerce/woocommerce_rms_normal/class-gateway.php (hunk 3 above)
Comment (syahruddin-syahrir) [reply WooCommerce/woocommerce_rms_normal/class-gateway.php]: Make it pretty : 639d7d75996fc23dcc2508b73c73328c707bf713

File: WooCommerce/woocommerce_rms_normal/class-gateway.php [hunk 4]
```diff
@@ -1,628 +1,874 @@
...
+        $this->description = $this->settings['description'];
+        $this->merchant_id = $this->settings['merchant_id'];
+        $this->verify_key = $this->settings['verify_key'];
+        $this->secret_key = $this->settings['secret_key'];
+        $this->account_type = $this->settings['account_type'];
+        $this->recurring = $this->settings['recurring'];
+
+        // Define hostname based on account_type
+        $this->url = ($this->get_option('account_type') == '1') ? "https://pay.fiuu.com/" : "https://sandbox.merchant.razer.com/";
+        $this->inquiry_url = ($this->get_option('account_type') == '1') ? "https://api.merchant.razer.com/" : "https://sandbox.merchant.razer.com/";
```
Comment (wankimmy) [inline WooCommerce/woocommerce_rms_normal/class-gateway.php]: change to api.fiuu.com

File: WooCommerce/woocommerce_rms_normal/class-gateway.php [hunk 5]
```diff
@@ -1,628 +1,874 @@
...
+                'title' => __('Description', 'wcmolpay'),
+                'type' => 'textarea',
+                'description' => __('This controls the description which the user sees during checkout.', 'wcmolpay'),
+                'default' => __('Pay with Fiuu', 'wcmolpay'),
+                'desc_tip' => true,
+            ),
+            'merchant_id' => array(
+                'title' => __('Merchant ID', 'wcmolpay'),
+                'type' => 'text',
+                'description' => __('Please enter your Fiuu Merchant ID.', 'wcmolpay') . ' ' . sprintf(__('You can to get this information in: %sFiuu Account%s.', 'wcmolpay'), '<a href="https://portal.merchant.razer.com/" target="_blank">', '</a>'),
```
Comment (wankimmy) [inline WooCommerce/woocommerce_rms_normal/class-gateway.php]: change to portal.fiuu.com

File: WooCommerce/woocommerce_rms_normal/class-gateway.php (hunk 4 above)
Comment (syahruddin-syahrir) [reply WooCommerce/woocommerce_rms_normal/class-gateway.php]: changed

proof: 013ce67a70abd17bd3027c6e2410e84c1b2c025a

File: WooCommerce/woocommerce_rms_normal/class-gateway.php (hunk 5 above)
Comment (syahruddin-syahrir) [reply WooCommerce/woocommerce_rms_normal/class-gateway.php]: changed 

proof: 013ce67a70abd17bd3027c6e2410e84c1b2c025a

**Pull request merged**